helm/korifi/gateway-api-router/manifests.yaml

```diff
@@ -0,0 +1,113 @@
+---
+apiVersion: admissionregistration.k8s.io/v1
+kind: MutatingWebhookConfiguration
+metadata:
+  name: korifi-gateway-api-router-mutating-webhook-configuration
+  annotations:
+    cert-manager.io/inject-ca-from: '{{ .Release.Namespace }}/korifi-gateway-api-router-serving-cert'
+webhooks:
+  - admissionReviewVersions:
+      - v1
+      - v1beta1
+    clientConfig:
+      service:
+        name: korifi-gateway-api-router-webhook-service
+        namespace: '{{ .Release.Namespace }}'
+        path: /mutate-korifi-cloudfoundry-org-v1alpha1-cfroute
+    failurePolicy: Fail
+    name: mcfroute.korifi.cloudfoundry.org
```

Comment:
This MutatingWebhookConfiguration is right now duplicated in both controllers and gateway-api-router, but as I suggested above we should move the webhooks back to controller and delete the webhook generation utilities from the router. Once we do that, there will no longer be a duplication.